src/osd/OSDMap.cc

```diff
@@ -2157,7 +2157,7 @@ bool OSDMap::check_pg_upmaps(
                        << j->first << " " << j->second
                        << dendl;
         to_cancel->push_back(pg);
-      } else {
+      } else if (newmap != j->second) {
         //Josh--check partial no-op here.
```

Comment:
Please remove the "Josh" from the comment - it was an internal mark for me, but it is not needed anymore (I forgot to fix it)

Reply:
Sure, will fix.

Reply:
Fixed.